docs/sources/developer/Developers-Guide.md

```diff
@@ -50,51 +36,74 @@ Overview of the general architecture of Plaso:
+Want to add a parser to Plaso and you are ready to go? Start by checking 
+[here](https://github.com/log2timeline/plaso/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) 
+if someone is already working on it. If you don't see anything there you can 
+just go ahead and [create an issue on the github site](https://github.com/log2timeline/plaso/issues) 
+and mark it as "enhancement". Assign the issue to yourself so that we can keep 
+track on who is working on what.
 
-If you cannot program and still have a great idea for a feature please go ahead and create an issue and leave it unassigned, note that the priority will be who ever wants to work on it. Or consider this the idea opportunity to learn yourself Python programming.
+If you cannot program and still have a great idea for a feature please go ahead 
+and create an issue and leave it unassigned, note that the priority will be who 
+ever wants to work on it.
 
-Before you start writing the code, please review the following:
+Before you start writing  code, please review the following:
 
-* [Style guide](Style-guide.md). All code submitted to the project needs to follow this style guide.
-* [Code review](Codereview.md). All code that is submitted into the project needs to be reviewed by at least one other person.
-* [Adding a new dependency](https://github.com/log2timeline/l2tdocs/blob/master/process/Dependencies.md). If your code requires adding a new dependency please check out these instructions.
+* [Style guide](Style-guide.md). All code submitted to the project needs to 
+follow this style guide.
+* [Code review](Codereview.md). All code that is submitted into the project is
+ reviewed by at least one other person.
+* [Adding a new dependency](https://github.com/log2timeline/l2tdocs/blob/master/process/Dependencies.md). 
+If your code requires adding a new dependency please check out these instructions.
 
 ### Before you submit your first code review
 
-1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/?fromgroups#!forum/log2timeline-dev), we recommend using the same account as step 1
-2. Install the required development tools like pylint, python-mock, sphinx
-3. Make sure to run all the tests inside the Plaso and the dfVFS codebase, and that they successfully complete on your development system
-4. Make sure your development system is set up correctly so that you can develop and test correctly.
-5. Make sure your email address and name are correctly set in git e.g.:
+1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/?fromgroups#!forum/log2timeline-dev),
```

Comment:
maybe mention the slack channel as well? you can reference https://github.com/open-source-dfir/slack

Reply:
Done